AppBar 외 5개 페이지 추가

Szzng opened the pull request:
추가된 디자인들
- AppBar + 로그인 dialog
- MainHome, ('/')
- 토픽 ('/topic') 
- 두근두근 후기 ('/topic/afterpitapat')
- 포스트 read page ('/topic/postdetail' - 추후 url 수정)
- 두근두근 ('/pitapat')

Szzng committed:
color, img, App.vue 세팅 및 불필요한 예제 파일 삭제

Szzng committed:
Installed @mdi/font -D and set to use mdi-icon

Szzng committed:
add AppBAr

Szzng committed:
add page "MainHome"

Szzng committed:
add pages "Topic"

Szzng committed:
add page "postDetail"

Szzng committed:
(ing)wireframe of page "pitapat"

Szzng committed:
router to history mode

Szzng committed:
wireframe of page pitapat

Szzng committed:
코드 정리 및 통일

Szzng committed:
change appbart topic url 'topic' to 'topic/afterpitapat'

Szzng committed:
쓸데없는 속성 지움

Szzng committed:
글쓰기 버튼 추가

teacherSsamko commented on src/App.vue:
```diff
@@ -1,13 +1,29 @@
-    <img src="./assets/logo.png">
-    <router-view/>
-  </div>
+  <v-app id="inspire">
```
이거는 왜 inspire야? 보통 메인 app은 app으로 하지않아? 그냥 이건 궁금해서

Szzng replied:
ㅎㅎ 아무 의도 없음... 코드펜에서 작성하면 저렇게 주던데 과거의 내가 그거 가져왔나봄 ㅋㅋㅋ app으로 수정해야겠다....

teacherSsamko commented on src/components/AppBar.vue:
```diff
@@ -0,0 +1,222 @@
+<template>
+  <div>
+    <!-- AppBar -->
```
이 주석은 없어도 될 듯!

teacherSsamko commented on src/components/AppBar.vue:
```diff
@@ -0,0 +1,222 @@
+      flat
+      style="border-bottom: 1px solid #d2d2d2 !important"
+    >
+      <!-- teameat logo img -->
```
이것도 없어도 되겠다!

teacherSsamko commented on src/components/AppBar.vue:
```diff
@@ -0,0 +1,222 @@
+      <!-- 검색창 -->
+      <v-responsive max-width="240">
+        <v-text-field
+          class="search_box"
```
search_box == 검색창

teacherSsamko commented on src/components/MainHome.vue:
```diff
@@ -0,0 +1,179 @@
+        comment_cnt: 100
+      }
+    ],
+    length: 3,
+    onboarding: 0
```
length랑 onboarding이 뭘 의미하는거야??

Szzng replied:
이거는 v-window 관련해서 필요한 변수들인데, 변수명에 window 추가할게~

teacherSsamko reviewed (commented):
좋은 코드는 주석이 없이도 이해가 가능한 코드! 굳이 넣지 않아도 알 수 있는 곳에 있는 주석은 다 빼버리자~

sendmethere committed:
Topic page details modified, style css created

Szzng committed:
설명용 주석 삭제

Szzng committed:
Merge branch 'szzng' of https://github.com/coding-kkanbu/teameet_front into szzng

Szzng committed:
v-window 관련 변수명에 'window'추가

Szzng committed:
v-app id를 inspire -> app 변경

teacherSsamko commented on src/router/index.js:
```diff
@@ -1,15 +1,36 @@
+  mode: 'history',
   routes: [
+    { path: '/', component: MainHome },
+    { path: '/topic', component: Topic },
+    { path: '/pitapat', component: PitAPat },
+    { path: '/topic/postdetail', component: PostDetail },
```
topic이 아래에 정의되어 있는데 왜 여기에도 같이 있죠????

Szzng replied:
안그래도 어제 인디스쿨 코드 보면서 이 부분 컴포넌트 하나로 바꿔야지 했는데 딱 걸렸넹ㅋㅋㅋ 내일 중으로 수정할게~!!

Szzng replied:
@teacherSsamko 하나의 파일로 합침!

teacherSsamko reviewed (commented):
`src/componennts/Topic/` 의 경우 각 토픽게시판이 제목만 다르고 api호출하는것만 다르고 나머지는 다 똑같을것같은데 이럴경우엔 따로따로 페이지를 만들어주기보다 하나의 템플릿을 재사용하는게 좋아요~!

Szzng committed:
topic 아래 subtopic 페이지들을 하나의 SubTopic.vue로 합치기

Szzng committed:
v-app id를 다시 Inspire로 변경 및 v-main 태그를 App.vue에서 선언함

1. App.vue의 v-app id를 app으로 바꾸니 여백과 정렬이 흐트러지는 문제가 발생하여  v-app id를 다시 inspire로 바꿈.
2. <v-main>태그를 App.vue에서 선언하고, 다른 component의 v-main 태그를 모두 div로 바꿈.

Szzng committed:
<style> 정리

Szzng committed:
두근두근 post detail 페이지 추가

teacherSsamko commented on src/router/index.js:
```diff
@@ -2,34 +2,26 @@ import Vue from 'vue'
-        { path: 'invest', component: Invest },
-        { path: 'ssum', component: Ssum },
-        { path: 'travel', component: Travel }
+        { path: ':subtopic', component: SubTopic }
```
여기는 앞에 `:` 콜론 왜 붙인거야? 오타인가?

Szzng replied:
동적 라우트 매칭 https://router.vuejs.org/kr/guide/essentials/dynamic-matching.html

_src/componennts/Topic/ 의 경우 각 토픽게시판이 제목만 다르고 api호출하는것만 다르고 나머지는 다 똑같을것같은데 이럴경우엔 따로따로 페이지를 만들어주기보다 하나의 템플릿을 재사용하는게 좋아요~!_

위의 말을 동적 라우트로 하나의 템플릿을 데이터만 다르게 쓰라는 걸로 이해했는데!
